import_export/admin.py

```diff
@@ -617,10 +617,14 @@ def get_export_data(self, file_format, queryset, *args, **kwargs):
         if not self.has_export_permission(request):
             raise PermissionDenied
 
-        data = self.get_data_for_export(request, queryset, *args, **kwargs)
+        data = self.get_data_for_export(
+            request,
+            queryset,
+            *args,
+            **kwargs,
+        )
         export_data = file_format.export_data(
-            data,
-            escape_formulae=self.should_escape_formulae,
+            data, **{"escape": True} if self.should_escape_formulae else {}
```

Comment:
I wonder why this is better to h ave an empty dictionary vs just
```
escape=self.should_escape_formulae 
```
Is that a peculiarity of the tablib interface? or because should_escape_formulae could be None?

Reply:
> Is that a peculiarity of the tablib interface? 

Yes, and this identified a bug.  I can only set 'escape=True' if the file format is XLSX, otherwise an error is raised.  I've corrected this.